dicoogle/src/main/resources/webapp/sass/components/about.scss

```
@@ -6,9 +6,15 @@
 }
 
 .gridAbout {
-  margin-top: 10px;
+  margin: 10px;
+  padding: 10px;
```

Playing around with these numbers, I got a slightly nicer and tighter look for this grid using the following:

```suggestion
  margin: 0px 10px 10px 0px;
  padding: 5px 10px;
```